src/components/gameCard/gameCard.js

```diff
@@ -38,10 +41,28 @@ class GameCard extends Component {
         img.src = src;
     }
 
+    loadLabelTag() {
+        const { idGame, gamesInformation } = this.props;
+        const gameLabelsData = gamesInformation.get('gamesLabels').filter(gameLabel => idGame === gameLabel.game_id);
```

Comment:
I suggest move the list of labels for each game to the action fill. This Will help us for filtering